### ISSUE TITLE
A11y_AzureDataStudio_Connect to a SQL Server_Usability: Screen Reader is not announcing the status of a server is connected successfully or not 

### ISSUE THREAD

**Issue by Gayathri1501** [microsoft/azuredatastudio#18975]
### GitHub Tags:
#A11yMAS; #A11yUsable; #A11yTCS; #A11ySev4; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #WCAG4.1.3; #Win32; #AINotInScope; #Status Messages; #Narrator. 

### Environment Details:
Windows Version: Windows 11 Enterprise
Application name: Azure Data studio

### Repro Steps:
1.Turn on Narrator
2. Launch Azure Data Studio.
3. Select the Connections icon (if not already selected) from left bar.
4. Hover over the 'Servers' header and select the 'New Connection' icon.
5. Provide server and windows authentication details, then click 'Connect'.
6. Validate in left menu that server has been successfully connected.
7. Verify status messages after the server is connected successfully is announced by narrator or not.
### Note:
Similar issues can be observed in the following in the following:

1. Delete.
2. Conformation.

### Actual Result:
Screen Reader is not announcing the status of a server is connected successfully or not.

### Expected Result:
Screen Reader should be announced the status of a server is connected successfully.

### Attachment:
![A11y_AzureDataStudio_New Deployment_Usability_](https://user-images.githubusercontent.com/99962579/162373155-4bcc4d2b-7830-4039-885b-62d568ba3339.gif)

**Comment by alanrenmsft** [comment]
bulk closing the usability level issues since we are not planning to work on it anytime soon.
